rfs_test/TEST_actioninfo_schema.py

```diff
@@ -35,12 +35,48 @@
 #associated Action, and shall be false if the parameter is not required (optional) to perform the
 #associated Action.
 ###################################################################################################
-
-'''
-Step 1: Try performing an action without the parameter.
-Step 2: If Step 1 failed and Required property is False, fail the assertion. 
-Step 3: Else, pass the assertion. 
-'''
+def Assertion_ACTI104(self,log) :
+    log.AssertionID = 'ACTI104'
+    assertion_status = log.PASS
+    log.assertion_log('BEGIN_ASSERTION', None)
+    relative_uris = self.relative_uris
+    authorization = 'on'
+    rq_headers = self.request_headers()
+
+    root_link_key = 'Systems/529QB9450R6'    # Not the correct method, how can I get an exsisting system. 
```

Comment:
Will need to see if the tool has a cache of systems to test against... If not, then the test will need to read the collection, and then pick a system from the response.